lib/route-enumeration.mjs

```
@@ -398,16 +400,26 @@ function discoverAppRoutes() {
+      let sitemap = true
+      try {
+        const source = fs.readFileSync(path.join(dir, pageEntry.name), 'utf8')
+        sitemap =
+          !/\bnoindex\s*:\s*true\b/.test(source) &&
+          !/\brobots\s*:\s*\{[\s\S]*?\bindex\s*:\s*false\b/.test(source)
+      } catch {
+        /* unreadable pages stay discoverable and sitemap-eligible */
+      }
       out.push({
         href: href === '/' ? '/' : href,
         title: segments.length === 0 ? 'FrankX' : titleFromSlug(segments[segments.length - 1]),
```

ESLint

[error] Use .at(-1) instead of [segments.length - 1]

(e18e/prefer-array-at)